lua/entities/gmod_wire_cpu.lua

```diff
@@ -6,6 +6,13 @@ ENT.WireDebugName	= "ZCPU"
 
 if CLIENT then return end -- No more client
 
+local cpu_max_frequency = 1400000
```

Comment:
So, one last thing I guess.
Get rid of the ConVar change callback (`cvars.AddChangeCallback` statement) and `cpu_max_frequency` variable.
And then just replace the usages of the `cpu_max_frequency` with `wire_cpu_max_frequency:GetInt()`.
Anyways, it looks good to me.